.github/workflows/ci.yml

```diff
@@ -0,0 +1,87 @@
+name: CI
+
+on:
+  push:
+    branches: [master, staging]
+  pull_request:
+    branches: [master, staging]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+env:
+  CHROME_BIN: /usr/bin/google-chrome
+
+permissions:
+  id-token: write
+  contents: write
+
+jobs:
+  test:
+    name: Checks & Build Test
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 12
+      - name: Install Dependenices
+        run: npm install
```

Comment:
we are actually using yarn and not npm in this repo

Reply:
Updated NPM to YARN.